src/Agent.Worker/Release/ReleaseTrackingManager.cs

```diff
@@ -0,0 +1,234 @@
+using Microsoft.TeamFoundation.DistributedTask.WebApi;
+using Microsoft.VisualStudio.Services.Agent.Util;
+using Microsoft.VisualStudio.Services.Agent.Worker.Build;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Globalization;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
+{
+    [ServiceLocator(Default = typeof(ReleaseTrackingManager))]
+    public interface IReleaseTrackingManager : IAgentService
```

Comment:
only `MarkExpiredForGarbageCollection` and `DisposeCollectedGarbage` are used outside this file
consider remove methods from this interface or refactor code to use them.

Reply:
removed unused methods